File: Python/8py 1/excluding_vat_price.py
"""
|--------------------------------------------------------------------------
| Calculate Price Excluding VAT
|--------------------------------------------------------------------------
|
"""

def excluding_vat_price(price):
   return round(price/1.15, 2) if price != None else -1 


def excluding_vat_price(price):
   try:
      return round(price / 1.15, 2)
   except TypeError:
      return -1


def excluding_vat_price(price):
   return -(not price) or round(price/1.15, 2)


print(excluding_vat_price(230.00), 200.00)
print(excluding_vat_price(123), 106.96)

File: PHP/8php 1/excludingVatPrice.php
<?php
/*
|--------------------------------------------------------------------------
| Calculate Price Excluding VAT
|--------------------------------------------------------------------------
|
*/

function excludingVatPrice($price){
   return $price != null ? number_format($price/1.15, 2) : -1;
}


function excludingVatPrice($price){  
   return round($price/1.15, 2) ?: -1;
}


function excludingVatPrice($price) {
   return $price ? round($price / 1.15, 2) : -1;
}


print_r(excludingVatPrice(230) . ' ' . 200.00 . '
');
print_r(excludingVatPrice(123) . ' ' . 106.96 . '
');
print_r(excludingVatPrice(null) . ' ' . -1 . '
');